baselines/hfedxgboost/README.md

```diff
@@ -0,0 +1,278 @@
+python -m hfedxgboost.main --config-name "centralized_basline_all_datasets_paper_config"
+
+#to run all the experiments for the centralized model
+python -m hfedxgboost.main --config-name "Centralized_Baseline"
+
+#to run the federated version for a9a dataset with 2 clients
+python -m hfedxgboost.main dataset="a9a" clients="a9a_2_clients"
+
+#to run the federated version for a9a dataset with 5 clients
+python -m hfedxgboost.main dataset="a9a" clients="a9a_5_clients"
+
+#to run the federated version for a9a dataset with 10 clients
+python -m hfedxgboost.main dataset="a9a" clients="a9a_10_clients"
+
+#to run the federated version for cod-rna dataset with 2 clients
+python -m hfedxgboost.main dataset="cod_rna" clients="cod_rna_2_clients"
+
+#to run the federated version for cod-rna dataset with 5 clients
+python -m hfedxgboost.main dataset="cod_rna" clients="cod_rna_5_clients"
+
+#to run the federated version for cod-rna dataset with 10 clients
+python -m hfedxgboost.main dataset="cod_rna" clients="cod_rna_10_clients"
+
+#to run the federated version for space_ga dataset with 2 clients
+python -m hfedxgboost.main dataset="space_ga" clients="space_ga_2_clients"
+
+
+#to run the federated version for space_ga dataset with 5 clients
+python -m hfedxgboost.main dataset="space_ga" clients="space_ga_5_clients"
+
+#to run the federated version for space_ga dataset with 10 clients
+python -m hfedxgboost.main dataset="space_ga" clients="space_ga_10_clients"
+
+# The main experiment implemented in your baseline using default hyperparameters (that should be set in the Hydra configs) should run (including dataset download and necessary partitioning) by executing the command:
+
+poetry run -m <baseline-name>.main <no additional arguments> # where <baseline-name> is the name of this directory and that of the only sub-directory in this directory (i.e. where all your source code is)
+
+# If you are using a dataset that requires a complicated download (i.e. not using one natively supported by TF/PyTorch) + preprocessing logic, you might want to tell people to run one script first that will do all that. Please ensure the download + preprocessing can be configured to suit (at least!) a different download directory (and use as default the current directory). The expected command to run to do this is:
+
+poetry run -m <baseline-name>.dataset_preparation <optional arguments, but default should always run>
+
+# It is expected that you baseline supports more than one dataset and different FL settings (e.g. different number of clients, dataset partitioning methods, etc). Please provide a list of commands showing how these experiments are run. Include also a short explanation of what each one does. Here it is expected you'll be using the Hydra syntax to override the default config.
+
+poetry run -m <baseline-name>.main  <override_some_hyperparameters>
+.
+.
+.
+poetry run -m <baseline-name>.main  <override_some_hyperparameters>
+```
+
+
+## Expected Results
+
+
```

Comment:
Are you planning to mention in this section the existence of `results.csv` and `results_centralized.csv`? or these are just temporal files that you use while working on your baseline?

Reply:
I mentioned them in the Training Hyperparameters section since they are related to the fine-tuning of the hyperparameters process.